.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
```

Comment:
I'm not sure I want X IDE settings files for each new dev on the project :thinking:. Maybe you can add in the .gitignore and put it as a suggestion in the README.md ? Something like "If you are using VSCode you can add `"python.formatting.provider": "black"` to your `.vscode/settings.json` file :thinking: